sources/packages/backend/libs/test-utils/src/data-source/e2e-data-source.ts

```diff
@@ -137,6 +138,7 @@ export function createE2EDataSources(dataSource: DataSource): E2EDataSources {
       ApplicationOfferingChangeRequest,
     ),
     studentLoanBalance: dataSource.getRepository(StudentLoanBalance),
+    ecertFeedbackError: dataSource.getRepository(ECertFeedbackError),
```

Comment:
If we are using `ECertFeedbackError` where `EC` is capital we should have `ecertFeedbackError` defined as `eCertFeedbackError`. Does it make sense?

Reply:
Yes it does and I will change.